### ISSUE TITLE
Is it 89 or 90 pairs of real training images?

### ISSUE THREAD

**Issue by xixi-nobody** [mingcv/DSRNet#7]
Both the paper you mentioned and the NIPS21 paper "Trash or Treasure? An Interactive Dual-Stream Strategy for Single Image Reflection Separation" refer to the use of 90 real pairs. However, whether it is the Baidu Pan or Google Drive link provided by you or the dataset division based on the paper "Single Image Reflection Removal with Perceptual Losses," there should be 89 pairs instead of 90 pairs. Could you please clarify why there is a mention of 90 pairs?

**Comment by mingcv** [comment]
Sorry for any misunderstanding caused. 

We followed the expression of the data division provided in the previous papers "Single Image Reflection Separation with Perceptual Losses," "Single Image Reflection Removal Exploiting Misaligned Training Data and Network Enhancements," and "Single Image Reflection Removal Through Cascaded Refinement" for both of our papers in case of any review problems. However, from the beginning, I worked on the reflection separation problem; the data link Zhang et al. provided contains only 89 real pairs for training. I cannot figure out why this discrepancy occurs, but I can ensure that all of our models are trained with the 89 real pairs.
